suite-native/intl/src/en.ts

```diff
@@ -688,6 +689,18 @@ export const en = {
             subtitle: 'Get started by receiving coins',
             button: 'Receive',
         },
+        detail: {
+            header: '<transactionType></transactionType> transaction',
```

Comment:
fyi: Recently i realised, that you can also use {}
```suggestion
            header: '{transactionType></transactionType> transaction',
```

and later in code:

``` values={{ transactionType: transaction.type }}```